bin/node/cli/Cargo.toml

```diff
@@ -30,7 +30,7 @@ hex-literal = "0.2.1"
 jsonrpc-core = "14.0.3"
 log = "0.4.8"
 rand = "0.7.2"
-structopt = "=0.3.7"
+structopt = "0.3.8"
```

Comment:
This was pinned because of https://github.com/paritytech/substrate/pull/4509
I don't know anything about that issue, but did we stop depending on structopt's internals?

Reply:
Yes it was because we were using internals of structopt. Part of my refactoring goal was also to remove that need and use only the public API so it can be upgraded easily.